app/models/abuse_report.rb

```diff
@@ -49,4 +65,23 @@ def send_report
     )
     reporter.send_report!
   end
+
+  # if the URL clearly belongs to a work (i.e. contains "works/123")
+  # make sure it isn't reported more than ABUSE_REPORTS_PER_WORK_MAX times per month
+  def work_is_not_over_reported
```

Comment:
Assignment Branch Condition size for work_is_not_over_reported is too high. [15.26/15]